src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerEndpointTest.java

```diff
@@ -296,7 +285,6 @@ void post_case_event_duplicate_message_on_dlq_should_update_delivery_count_when_
         @Test
         void dlq_case_event_message_should_be_stored_and_return_200_ok() throws Exception {
 
-            when(launchDarklyFeatureFlagProvider.getBooleanValue(any(), any())).thenReturn(true);
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.